pkg/model/resource/utils.go

```diff
@@ -50,6 +50,17 @@ func safeImport(unsafe string) string {
 
 // APIPackagePath returns the default path
 func APIPackagePath(repo, group, version string, multiGroup bool) string {
```

Comment:
Would it have been an easier transition to add the `isLegacyLayout` boolean to this function signature and handle it that way instead of creating separate functions?

Reply:
I though better keep both to after we are able to remove the legacy one instead